sdk/js/src/utils/consts.ts

```diff
@@ -18,6 +18,8 @@ export const CHAINS = {
   moonbeam: 16,
   neon: 17,
   terra2: 18,
+  aptos: 20,
```

Comment:
I think 20 might be reserved for Osmosis but the PR hasn’t landed yet

Reply:
Then I win?  :)

Reply:
Updated to 22